integration_tests/tests/flowapi_tests/test_api_spec.test_generated_openapi_json_spec.approved.txt

```diff
@@ -0,0 +1,1111 @@
+          },
+          "locations": {
+            "items": {
+              "$ref": "#/components/schemas/InputToModalLocation"
+            },
+            "minItems": 1,
+            "type": "array"
+          },
+          "query_kind": {
+            "enum": [
+              "modal_location"
+            ],
+            "type": "string"
+          },
+          "subscriber_subset": {
+            "enum": [
+              null
+            ],
+            "nullable": true,
+            "type": "string"
+          }
+        },
+        "required": [
+          "aggregation_unit",
+          "query_kind"
+        ],
+        "type": "object"
+      },
+      "SpatialAggregate": {
+        "properties": {
+          "locations": {
+            "$ref": "#/components/schemas/InputToSpatialAggregate"
+          },
+          "query_kind": {
+            "enum": [
+              "spatial_aggregate"
+            ],
+            "type": "string"
+          }
+        },
+        "required": [
+          "locations",
+          "query_kind"
+        ],
+        "type": "object"
+      },
+      "TotalNetworkObjects": {
+        "properties": {
+          "aggregation_unit": {
+            "enum": [
+              "admin0",
+              "admin1",
+              "admin2",
+              "admin3"
+            ],
+            "type": "string"
+          },
+          "end_date": {
+            "format": "date",
+            "type": "string"
+          },
+          "period": {
+            "enum": [
+              "day",
+              "hour",
+              "minute",
+              "month",
+              "second",
+              "year"
+            ],
+            "type": "string"
+          },
+          "query_kind": {
+            "enum": [
+              "total_network_objects"
+            ],
+            "type": "string"
+          },
+          "start_date": {
+            "format": "date",
+            "type": "string"
+          }
+        },
+        "required": [
+          "aggregation_unit",
+          "end_date",
+          "query_kind",
+          "start_date"
+        ],
+        "type": "object"
+      },
+      "UniqueSubscriberCounts": {
+        "properties": {
+          "aggregation_unit": {
+            "enum": [
+              "admin0",
+              "admin1",
+              "admin2",
+              "admin3"
+            ],
+            "type": "string"
+          },
+          "end_date": {
+            "format": "date",
+            "type": "string"
+          },
+          "query_kind": {
+            "enum": [
+              "unique_subscriber_counts"
+            ],
+            "type": "string"
+          },
+          "start_date": {
+            "format": "date",
+            "type": "string"
+          }
+        },
+        "required": [
+          "aggregation_unit",
+          "end_date",
+          "query_kind",
+          "start_date"
+        ],
+        "type": "object"
+      }
+    }
+  },
+  "info": {
```

Comment:
When I paste this file into the swagger editor, it complains about a missing version:
```
Structural error at info
should have required property 'version'
missingProperty: version
```

Reply:
Version is (at the minute) defined by `flowapi.__version__`, which varies by commit because of versioneer, so I've stripped it out of the ApprovalTests bit.